src/main/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadSVG.java

```diff
@@ -367,7 +367,7 @@ private void doCubicCurveAbs(SVGPathSeg item, Matrix3d m) {
 		ArrayList<Point2D> points = b.generateCurvePoints(0.1);
```

Comment:
```suggestion
		List<Point2D> points = b.generateCurvePoints(0.1);
```

Reply:
why is it better to use generic List?  One extra include....

Reply:
you don't need the method specific to an ArrayList in this part of the code, you just need a List which is the interface.

Reply:
I don't need an extra include, either!  If I had a factory that was returning a List then sure, generic allows for more flex.  But here it's of no benefit.

Reply:
`import` are generated by the IDE, so it is not an issue. your main problem is why ArrayList over List. I may not using the right word to explain so, instead, I suggest you to read this small answer: https://stackoverflow.com/questions/147468/why-should-the-interface-for-a-java-class-be-preferred